docs-source/cloudbank/content/provision/install-free-trial.md

```diff
@@ -148,3 +148,72 @@ You can also use this approach in any regular commercial Oracle Cloud Tenancy -
    * The Spring Operations Center main dashboard will appear similar to this:
 
      ![Spring Operations Center](../images/install-free-tier-5.png " ")
+
+1. DB Access & setup
+
+   * Connect to your instance (using SSH), find the database pod with the following command: `kubectl get pods --all-namespaces`.
+
+      In the `oracle-database-operator-systemthen the address` namespace, look for a pods like `free-XXXX`.
+
+   * Access to DB instance with the following command: `kubectl exec -it  -n oracle-database-operator-system free-XXXX -- /bin/sh`
+
+   * Login as sysdba to create the user `account` that will be used in Module 3:  
+
+      ```sql
+      sqlplus / as sysdba
+      ALTER SESSION SET CONTAINER = freepdb1;    
+      CREATE TABLESPACE users DATAFILE '/opt/oracle/oradata/FREE/FREEPDB1/users.dbf'  SIZE 100M AUTOEXTEND ON NEXT 10M MAXSIZE 500M;
+      CREATE USER account IDENTIFIED BY Welcome1234## DEFAULT TABLESPACE users TEMPORARY TABLESPACE temp;
+      ```
+
+      This step replaces the steps from 1 to 4 in the task **Account Microservices/Prepare Database Objects**.
+
+      In Step 5 of **Prepare Database Objects**, skip the command: `create user account identified by "Welcome1234##";` and execute all the rest, since the user account has been already created.
+
+      If you need to access the db from an ssh connection, these are the commands to get `username` and `password`:
+
+      ```bash
+      #to get <UID>:
+      kubectl -n application get secret baas-db-secrets -o jsonpath='{.data.db\.password}' | base64 -d
+      #to get <PWD>:
+      kubectl -n application get secret baas-db-secrets -o jsonpath='{.data.db\.username}' | base64 -d
+      #for pl/sql:
+      kubectl exec -n oracle-database-operator-system free-tfl4e -ti -- sqlplus <UDI>/<PWD>@localhost:1521/freepdb1
+      ```
+
+1. Kubernetes access from own desktop
+
+    * On the compute node (logged in using SSH), execute the following command:
+
+        ```sh
+        kubectl config view --minify --context=default --flatten > /home/ubuntu/context-config.yaml
+        ```
+
+    * Download the YAML configuration file to your local machine:
+
+        ```sh
+        scp -i <YOUR_PRIVATE_KEY.key> ubuntu@<VM_IP>:/home/ubuntu/context-config.yaml <YOUR_DIR>/context-config.yaml 
+        ```
+
+    * In the `context-config.yaml` locate the `server` entry:
+
+        ```yaml
+        ...
+        - cluster:
+          certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJlRENDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdGM...
+          server: https://127.0.0.1:6443
+        name: default
+        ...
+        ```
+
+        and change with <YOUR_DIR>.
```

Comment:
Not sure I understand ` and change with <YOUR_DIR>.`

Reply:
fixed